mbuild/tests/test_gmso.py

```diff
@@ -56,13 +56,17 @@ def test_infer_hier(selfs, ethane):
         # Create an ethane box, should be a four structure
         eth_box = mb.packing.fill_box(compound=ethane, n_compounds=1, density=1)
 
+        # TODO: once infer_hierarchy is implemented in gmso, changes these functions
```

Comment:
I think you can pass the kwarg `parse_label=False` and that will do the trick. I think the label `infer_hierachy` is equivalent to the `parse_label`, where default value is True. So unless we want to rename this argument, then I'm not quite sure that we need the TODO note here.

Reply:
I made the suggested change.